infra/bridge/src/utils/index.js

```diff
@@ -44,10 +44,32 @@ function getAbsoluteUrl(relativeUrl, params = {}) {
   return url.toString()
 }
 
+const htmlEntities = {
+  amp: '&',
+  apos: "'",
+  lt: '<',
+  gt: '>',
+  quot: '"',
+  nbsp: '\xa0'
+}
+
+const htmlEntityPattern = /&([a-z]+);/gi
+
+function decodeHTML(content) {
```

Comment:
nit: it's fairly trivial code but would be good to add a unit test... just in case :) Also helps with catching regressions in the future. I would use as input the tweet content from anna on [this doc](https://docs.google.com/spreadsheets/d/1akbpgFFy5vW-PszmcJNImIW2rjkiwDk4xVXlUO7fMzo/edit#gid=0).

Reply:
Sounds good. Will add the tests.